package.json

```diff
@@ -11,7 +11,8 @@
     "localStart": "nodemon ./server/server.js",
     "build": "cd client && npm run build",
     "dev": "npm run client && npm run localStart",
-    "heroku-postbuild": "cd client && npm install && npm run build"
+    "heroku-postbuild": "cd client && npm install && npm run build",
+    "db:query": "psql -U staff -d ydrc1 ./queries.sql"
```

Comment:
You have to add file path after symbol ```-f``` 
The script should be ```psql -U 'database_username' -d 'database_name' -a -f 'file_path' ```

Reply:
done :+1: